janus.c

```diff
@@ -3467,19 +3468,20 @@ gboolean carbyne_janus_transport_is_sanityhealthcheck_resources_available(janus_
 	FILE *sgwRtspStatusFile = NULL;
     char * sgwline = NULL;
     size_t sgwlen = 0;
-    ssize_t sgwread;
-	sgwRtspStatusFile = fopen("/home/ubuntu/sgw-rtsp-status", "r");
+    ssize_t sgwStatusFileRead=0;
+	sgwRtspStatusFile = fopen(SGW_STATUS, "r");
 	if (sgwRtspStatusFile == NULL){
 			JANUS_LOG(LOG_ERR,"Failed opening sgw status file\n");
     		return FALSE;
 	}
-	while ((sgwread = getline(&sgwline, &sgwlen, sgwRtspStatusFile)) != -1) {
+	while ((sgwStatusFileRead = getline(&sgwline, &sgwlen, sgwRtspStatusFile)) != -1) {
```

Comment:
1. Why do you need `sgwStatusFileRead`? you don't use it anywhere.
2. There is a memory leak in this code - you must free the memory of `sgwline`
Call `free(sgwline)`
3. Why do you need to do it a `while`? just read the first line that is supposed to contain "true" or "false"
4. This way you can check the return value ( != 1 ) and return accordingly.